docs/apps/lnw/es2k/es2k-lnw-overlay-vms.md

```diff
@@ -126,7 +113,7 @@ Note: Here VSI 9 has been used as one of the ACC port representers and added to
 
 ### Start OvS as a separate process
 
-Legacy OvS is used as a control plane for source MAC learning of overlay VM's. OvS should be started as a separate process.
+Enhanced legacy OvS is used as a control plane for source MAC learning of overlay VM's. This OvS binary is available as part of ACC build and should be started as a separate process.
```

Comment:
I am not sure its clear what `Enhanced legacy OvS` means.

We are probably referring to our P4 enhancement "sidecar" to goes with OvS right? If yes, we should just call it "Enhanced OvS"